css/details.css

```diff
@@ -12,13 +24,143 @@ th {
 }
 
 td {
-	text-align: right;
+	text-align: left;
+	padding-left: 1em;
+	max-width: 4em;
 }
 
 summary {
 	cursor: pointer;
+	margin-bottom: 1em;
+	user-select: none;
+}
+
+/* Imprecise rows */
+
+tr.imprecise {
+	font-style: italic;
+	z-index: 10;
+}
+
+tr.imprecise td {
+	position: relative;
+}
+
+/* Question mark circle icon */
+tr.imprecise td::after {
+	content: "?";
+	position: absolute;
+	top: 0.65em;
+	right: -1.2em;
+	width: 1.2em;
+	height: 1.2em;
+	border-radius: 1em;
+	border: 1px solid #333;
+	font-size: 90%;
+	font-style: normal;
+	color: #000;
+	line-height: 15px;
+	font-weight: bold;
+	text-align: center;
+	background: #fff;
+	cursor: pointer;
+}
+
+
+/* Explanation tooltip for cotisation lines */
+
+.imprecise td aside {
+	position: absolute;
+	z-index: 11;
+	left: 40%;
+	width: 35em;
+	padding: 10px 15px;
+	margin-bottom: 5px;
```

Comment:
Once again, please don't use `px` values. This `margin` value should be relative to the text size.